ow_dynamic_terrain/scripts/modify_terrain_grinder_pub.py

```diff
@@ -19,6 +18,7 @@
 class ModifyTerrainGrinder:
 
   def __init__(self, *args):
+    rospy.set_param('gazebo/use_sim_time', True)
```

Comment:
Is this really a parameter that is consumed by gazebo? Do you have any reference on this?

Reply:
I wrote about this here: https://babelfish.arc.nasa.gov/confluence/display/OCEANWATERS/Increase+Real+Time+Factor 

Here is the reference:  http://wiki.ros.org/Clock

Reply:
Damiana, my question was specifically on the use of **"gazebo/use_sim_time"**. Is this an actual topic that Gazebo makes use? I couldn't see any info on this topic besides your own confluence doc.

Reply:
Here is a $ rosparam list:
![image](https://user-images.githubusercontent.com/51930968/101947245-04235880-3ba5-11eb-984f-f0eafc0e4ea2.png)

Reply:
This parameter is not found on _melodic-devel branch_. If you are on the _increase_real_time_factor branch_ then this param would just show up because we are introducing it to the simulation as part of this statement `rospy.set_param('gazebo/use_sim_time', True)` (here and in other python nodes).